config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+
+  root 'works#main'
+  resources :users
```

Comment:
Do you need the full CRUD routes for users?  You're not using `:edit` or `:destroy` for example.